meta/table.go

```diff
@@ -118,78 +122,97 @@ func (t Table) Encode(list ListEncoder) [][]string {
 	return list.encode()
 }
 
-type Database interface {
-	Drop(Table) []string
-	Create(Table) []string
-	Insert(Table, ListEncoder) []string
+// TableList pairs together a Table and a ListEncoder.
+type TableList struct {
+	Table Table
+	List  ListEncoder
 }
 
-func Init(d Database, table Table, list ListEncoder) []string {
-	var res []string
-	for _, v := range d.Drop(table) {
-		if v == "" {
-			continue
-		}
-		res = append(res, v)
+// KeyValue contains key value pairs and their labels.
+type KeyValue struct {
+	label   string
+	desc    string
+	entries map[string]string
+}
```

Comment:
question: (more for my own learning) is this to enable easier creation of a Table directly from a (key-value) CSV source, do you don't need to create it's own `network.go / mark.go etc` type thing?

Reply:
The next PR should illustrate an example of this usage. But it's aimed more at data/information that is stored within delta but not as csv files.  So, the example is pulling in little XML "snippets" which are used to describe instrument responses without the complexity of having to model that information. These tend to end up being more akin to a key value store which I felt was a good fit for the name.